ui/src/views/NotAuthorized.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <page-content>
+    <h2 class="title">
+      Missing Authorization
+    </h2>
+    <p>
+      You have not the right to access this site. Please request access it via
+      <a :href="link"> {{ link }}</a>
+    </p>
```

Comment:
We could use the `details` provided by the backend in the error response. But I'm also fine with this.

Reply:
I was hesitating about this. The link is read from the error response